Author: <csmall@debian.org>
Description: ps.1 normal indent widths
pri field explained 
--no-heading comment is an alias for --no-headers
Removed setproctitle(1) reference 
Bug-Debian: http://bugs.debian.org/465761
Bug-Debian: http://bugs.debian.org/529045
Index: b/ps/ps.1
===================================================================
--- a/ps/ps.1	2009-11-24 20:53:06.000000000 +1100
+++ b/ps/ps.1	2009-11-24 21:00:31.000000000 +1100
@@ -18,19 +18,13 @@
 .nh
 .if n .ss 12 0
 .\"
-.\" See /usr/share/groff/current/tmac/an-old.tmac for what these do.
-.\" Setting them to zero provides extra space, but only do that for
-.\" plain text output. PostScript and such will remain indented.
-.if n .nr IN 0n
-.if n .nr an-prevailing-indent 0n
-.\"
 .\"
 .\" ColSize is used for the format spec table.
 .\" It's the left margin, minus the right, minus
 .\" the space needed for the 1st two columns.
 .\" Making it messy: inches, ens, points, scaled points...
 .\"
-.nr ColSize ((\n(.lu-\n(.iu/\n(.Hu-20u)n)
+.nr ColSize ((\n(.lu-\n(.iu/\n(.Hu-26u)n)
 .\"
 .\" This is for command options
 .nr OptSize (16u)
@@ -499,7 +493,7 @@
 .opt c
 Show the true command name. This is derived from the name of the
 executable file, rather than from the argv value. Command arguments
-and any modifications to them (see\ \fIsetproctitle\fR(3)) are
+and any modifications to them  are
 thus not shown. This option
 effectively turns the \fBargs\fR format keyword into the \fBcomm\fR
 format keyword; it is useful with the \fB\-f\fR format option and with
@@ -588,7 +582,8 @@
 repeat header lines, one per page of output
 
 .opt \-\-no\-headers
-print no header line at all
+print no header line at all. \-\-no\-heading is an alias for this
+option.
 
 .opt \-\-lines \ n
 set screen height
@@ -1159,6 +1154,10 @@
 parent process ID.
 T}
 
+pri	PRI	T{
+priority of the process. Higher number means lower priority
+T}
+
 psr	PSR	T{
 processor that process is currently assigned to.
 T}
